htdocs/takepos/pay.php

```diff
@@ -131,6 +131,12 @@ function addreceived(price)
     	<?php
     	if (empty($conf->global->TAKEPOS_NUMPAD)) print 'received+=String(price);'."\n";
     	else print 'received+=parseFloat(price);'."\n";
+		$constforcompanyid = 'CASHDESK_ID_THIRDPARTY'.$_SESSION["takeposterminal"];
+		if ($invoice->id > 0 && ($invoice->socid != $conf->global->$constforcompanyid)) {
```

Comment:
Why having a different behavior between the default thirdparty (that is similar to an anonymous customer) and another one ?